packages/manager/apps/pci-private-network/src/hooks/useGatewayAvailabilityRegion/useGatewayAvailabilityRegion.tsx

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from 'react';
+import { useProject, useProductAvailability } from '@ovh-ux/manager-pci-common';
+
+export default function useGatewayAvailabilityRegion(
+  region: string,
+  planCode: string,
+): boolean {
+  const { data: project } = useProject();
+  const { data } = useProductAvailability(project.project_id, {
+    planCode,
+  });
+
+  return useMemo(() => {
+    const gateway = data?.plans.find(({ code }) => code === planCode);
+
+    if (gateway) {
+      return gateway.regions.some(({ name }) => name === region);
+    }
+
+    return false;
```

Comment:
return type of hooks is boolean here

`export default function useGatewayAvailabilityRegion(
  region: string,
  planCode: string,
): boolean`

plus it will return the result of this `gateway.regions.some(({ name }) => name === region)`

Maybe the hook name confused ?